packages/df-blogs/src/ViewPost.tsx

```diff
@@ -65,6 +69,24 @@ function ViewPostInternal (props: ViewPostProps) {
     </>;
   };
 
+  const renderCommentPreviews = () => {
```

Comment:
Doesn't make sense to render previews of comments for now. See my comment above. Let's just render a text of a comment. That's why I suggest renaming the method: `renderCommentPreviews` -> `renderCommens`
```suggestion
  const renderComments = () => {
```